src/Avalonia.Base/Input/PinchEventArgs.cs

```diff
@@ -4,20 +4,32 @@ namespace Avalonia.Input
 {
     public class PinchEventArgs : RoutedEventArgs
     {
-        public PinchEventArgs(double scale, Point scaleOrigin) :  base(Gestures.PinchEvent)
+        public PinchEventArgs(double scale, Point scaleOrigin) : base(Gestures.PinchEvent)
         {
             Scale = scale;
             ScaleOrigin = scaleOrigin;
         }
 
+        public PinchEventArgs(double scale, Point scaleOrigin, double degree) : base(Gestures.PinchEvent)
```

Comment:
`double degree` should really be `double angle` to follow standard parameter naming conventions. Degrees would then be in comments.